packages/web/src/components/avatar/Avatar.tsx

```diff
@@ -25,12 +26,12 @@ const messages = {
   profile: 'profile'
 }
 
-type AvatarProps = {
+type AvatarProps = Omit<HarmonyAvatarProps, 'src'> & {
   userId: Maybe<ID>
 }
 
 export const Avatar = (props: AvatarProps) => {
-  const { userId } = props
+  const { userId, ...passthroughProps } = props
```

Comment:
can we norm with "...other"? im down to switch to something else eventually, but lets have it all consistent so easy to search and replace